### ISSUE TITLE
Version update

### ISSUE THREAD

**Issue by edimusrex** [Arelle/EdgarRenderer#5]
Seems like the SEC is running version 3.3.1.900 but the only available version from here and arelle.org is version 3.3.0.814.  We're running into some strange errors trying to preview 2016 projects and would like to test with the latest version to see if it resolves it.

**Comment by andrewperkins-wf** [comment]
@edimusrex I believe that the current code the SEC is running is on a branch, but I am unable to remember which one off hand.  @hefischer do you remember?

**Comment by hefischer** [comment]
the version numbers are out of sync at SEC between previewer and EDGAR.  Currently running EDGAR is edgr154 branch.  Previewer will go to edgr1601 a week ahead of EDGAR production going to edgr1601, and I'm not sure if EDGAR version number is tracking correctly to previewer version number

**Comment by slivelyrrd** [comment]
Does this mean if one sends a filing to EDGAR which is 2016 UGT based it
will have 6.22 errors? Currently the Previewer log does not show 6.22
errors on 2016 UGT based files, assuming because it is using edgr1601 but
if you load a set of 2016 UGT based files to edgr154 it does show 6.22
errors.

On Thu, Apr 7, 2016 at 8:41 AM, Herman Fischer <notifications@github.com>
wrote:

> the version numbers are out of sync at SEC between previewer and EDGAR.
> Currently running EDGAR is edgr154 branch. Previewer will go to edgr1601 a
> week ahead of EDGAR production going to edgr1601, and I'm not sure if EDGAR
> version number is tracking correctly to previewer version number
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Arelle/EdgarRenderer/issues/5#issuecomment-206960739>
>



-- 

*Sean Lively*
Product Owner - XBRL Solutions

RR Donnelley Financial Services
M: 206.792.6440

*sean.c.lively@rrd.com <ashu.rawat@rrd.com>*
*activedisclosure.com <http://www.activedisclosure.com/>*

                <https://www.facebook.com/RRDActiveDisclosure>
<https://twitter.com/activedisc> <https://www.linkedin.com/company/5218445>

**Comment by hamscher** [comment]
Of course.On April 7, 2016 at 11:41 AM Herman Fischer <notifications@github.com> wrote:the version numbers are out of sync at SEC between previewer and EDGAR. Currently running EDGAR is edgr154 branch. Previewer will go to edgr1601 a week ahead of EDGAR production going to edgr1601, and I'm not sure if EDGAR version number is tracking correctly to previewer version number—You are receiving this because you are subscribed to this thread.Reply to this email directly or view it on GitHub
